web/src/app/rotations/RotationForm.tsx

```diff
@@ -163,6 +173,12 @@ export default function RotationForm(props: RotationFormProps): JSX.Element {
               )
             }
           />
+          {handoffWarning && (
+            <Typography variant='body2' className={classes.handoffWarning}>
```

Comment:
you can apply styles directly now and I believe is the recommended way moving forward. it also comes with the theme object available as a callback function for every style:

```suggestion
            <Typography variant='body2' sx={{ color: (theme) => theme.palette.warning.main }}>
```